src/queries/GetSystemIntakeQuery.ts

```diff
@@ -155,6 +155,21 @@ export const SystemIntake = gql`
       decisionState
       submittedAt
     }
+    grbPresentationLinks {
```

Comment:
The `transcriptLink` field needs to be added to the query and displayed in the same place as the `transcriptFileURL` field on the card.